probe-rs/src/config/chip_info.rs

```diff
@@ -3,10 +3,10 @@ use crate::architecture::arm::ArmChipInfo;
 /// Information about a chip which is used
 /// for automatic detection of the connected chip.
 ///
-/// For ARM-based chips, the function [ArmProbeInterface::read_from_rom_table] is
+/// For ARM-based chips, the function [ArmProbeInterface::read_chip_info_from_rom_table][r] is
```

Comment:
You're inconsistent here with the rest of the function links. For one, other links are in backticks, and for two, you appended `()` to a few links.

Reply:
Well, I only fixed the broken links, for some adding the `()` fixed the target not found issue. I don’t know why it wasn’t flaged for this one, let me check 👍 

The backticks on the others were already there, but I can add them here too.

Reply:
Eh, don't bother, I'm not sure if locally updating a few is the way to go - we should maybe pick a style and go through the whole thing at once at one point in the future.

Reply:
Actually, the `()` are not necessary, it was the `Self::` part (in `riscv::communication_interface`).